Makefile

```diff
@@ -68,22 +68,21 @@ lint: ## Run code lint checks
 
 ##@ Tests
 
-.PHONY: test
+.PHONY: go-test
 test: ## Run the go tests ($ go test -v ./cmd/... ./pkg/...)
 	go test -v ./cmd/... ./pkg/...
 
-.PHONY: test-ci
-test-ci: ## Run the unit tests (used in the CI)
-	./setup.sh
+.PHONY: test
+test: ## Run the unit tests (used in the CI)
 	./test.sh
 
-.PHONY: test-e2e
-test-e2e: ## Run the integration tests (used in the CI)
-	./test_e2e.sh
-
 .PHONY: test-coverage
 test-coverage:  ## Run coveralls
 	# remove all coverage files if exists
 	- rm -rf *.out
 	# run the go tests and gen the file coverage-all used to do the integration with coverrals.io
 	go test -failfast -tags=integration -coverprofile=coverage-all.out -covermode=count ./pkg/... ./cmd/...
+
+.PHONY: test-gen
+test-gen: ## Run the script to ensure that the testdata is updated
```

Comment:
I think this should be `test-golden` since it's calling a script called `golden_test` :sunglasses:

Reply:
I rename the script and target to make more sense.